pkg/ovncontroller/daemonset.go

```diff
@@ -62,6 +62,32 @@ func CreateOVNDaemonSet(
 		}...)
 	}
 
+	ovnControllerLivenessProbe := &corev1.Probe{
+		// TODO might need tuning
+		TimeoutSeconds:      5,
+		PeriodSeconds:       5,
+		InitialDelaySeconds: 30,
+	}
+
+	ovnControllerReadinessProbe := &corev1.Probe{
+		// TODO might need tuning
+		TimeoutSeconds:      5,
+		PeriodSeconds:       5,
+		InitialDelaySeconds: 10,
```

Comment:
Noticing the difference in Iniitaldelaysecond curious on how these were computed? normally ready time > live time but here it's opposite.

Reply:
These were initial values, as noted in the TODO, and they may need tuning. The reason I set the livenessProbe to 30s is to ensure that the container has enough time to start before the liveness check can potentially cause a restart of the pod. The readinessProbe is less critical, as it would, at worst, leave the container unhealthy. However, upon further reflection, I don’t see the sense in having readiness checks before the first liveness check, so I’ll set it to the same value! We will have time to tune them in a future PR if necessary.